store/src/java/com/zimbra/cs/account/soap/SoapProvisioning.java

```diff
@@ -2949,4 +2949,8 @@ public void refreshUserCredentials(Account account) {
         throw new UnsupportedOperationException("Currently no way to refresh required attributes over SOAP");
     }
 
+    @Override
```

Comment:
This default impl should go in provisioning class

Reply:
implemented in Provisioning class.